lib/package/commands.coffee

```diff
@@ -30,6 +30,14 @@ module.exports =
         @withInk ->
           boot()
           juno.runtime.evaluation.toggleMeta 'methods'
+      'julia-client:reset-workspace': =>
+        if juno.connection.client.isActive()
+          editor = atom.workspace.getActiveTextEditor()
+          atom.commands.dispatch atom.views.getView(editor), 'inline-results:clear-all'
```

Comment:
Yea if it just scrolled the console to the bottom that would be fine but not good deleting error messages